Lib/test/test_pickle.py

```diff
@@ -483,6 +483,10 @@ def test_exceptions(self):
         self.assertEqual(reverse_mapping('builtins', 'OSError'),
                          ('exceptions', 'OSError'))
 
+        exceptions_not_in_python_2 = [
```

Comment:
IIUC, this test is checking a mapping between python2 and python3 exception types that pickle uses. So I thought I'd future proof it in case more exception classes are added.

Reply:
Ah, I totally missed that. But doesn't the block starting with `if exc in (BlockingIOError,` below serve the same purpose? Those all seem to be exceptions that don't exist in Python 2 and which are therefore skipped. Why not just add your exceptions there?